src/app/shared/pipes/application-filter.pipe.ts

```diff
@@ -8,7 +8,9 @@ import { Application } from '../models/application.model';
 export class ApplicationFilterPipe implements PipeTransform {
 
   transform(array: Application[], status: string): Application[] {
-    return array.filter(card => card.status === status);
+    // return array.filter(card => card.status === status);
```

Comment:
if you leave commented code please use TODO: for letting other teammates know that it is needed